.drone.star

```diff
@@ -3,7 +3,8 @@ def main(ctx):
     linting(ctx),
     unitTests(ctx),
     apiTests(ctx, 'master', 'f7bf41b725b8dac55748c1a090c0d6b3617c89e5'),
-    acceptanceTests(ctx, 'master', 'f9a0874dc016ee0269c698914ef3f2c75ce3e2e6'),
+    acceptanceTests(ctx, 'phoenixWebUI1', 'master', 'f9a0874dc016ee0269c698914ef3f2c75ce3e2e6'),
+    acceptanceTests(ctx, 'phoenixWebUI2', 'master', 'f9a0874dc016ee0269c698914ef3f2c75ce3e2e6'),
```

Comment:
The list of names of acceptance test suite sections to run could go in a `config[]` section. Then some new function can loop through the list and call `acceptanceTests()` to generate each pipeline, and the later `depends_on` sections can also get those from the same config section.